CHANGELOG.md

```diff
@@ -32,6 +32,183 @@
 - [v0.1.0-rc2](#v010-rc2)
 - [v0.1.0-rc1](#v010-rc1)
 
+# v1.0
+
+On behalf of Kubernetes SIG Network, we are pleased to announce the v1.0 release!
+This release marks a huge milestone for this project. Several key APIs are
+graduating to GA (generally available), while other significant features have
+been added to the Experimental channel.
+
+It's been four years since this project began, and we would never have got here
+without the support of a dedicated and active community. The maintainers would like
+to thanks _everyone_ who's contributed to Gateway API, whether in the form of
+commits to the repo, discussion, ideas, or general support.
+
+We literally couldn't have got this far without you.
+
+This project is nowhere near finished, as you can see from the large amount of
+features being added into Experimental.
+
+With such a big set of things still to do, contributors and contributions are
```

Comment:
```suggestion
features being added into Experimental. With such a big set of things still to do,
contributors and contributions are
```

Reply:
Maybe say "Experimental Channel" just to be extra clear for people new to the project.